Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -1111,6 +1121,45 @@ void TextEditor::unindent_line()
     }
 }
 
+void TextEditor::uppercase_selection()
+{
+    if (!has_selection())
+        return;
+    auto const selection_start = m_selection.start() > m_selection.end() ? m_selection.end() : m_selection.start();
+    auto const selection_end = m_selection.end() > m_selection.start() ? m_selection.end() : m_selection.start();
+
+    auto const range = TextRange(selection_start, selection_end);
```

Comment:
We already have the current selection handy:
```suggestion
    auto const range = selection();
```

(Same in `lowercase_selection()` and `snakecase_selection()`.)

Reply:
The changes requested above have been done. For the snake_selection I did update the selected text as the snake case will be larger than the original text.